presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/aggregation/ConvexHullAggregation.java

```diff
@@ -56,8 +56,10 @@ public static void input(@AggregationState GeometryState state,
             state.setGeometry(geometry.convexHull());
         }
         else if (!geometry.isEmpty()) {
+            state.addMemoryUsage(-state.estimateGeometryMemorySize());
```

Comment:
@tdcmeehan I find this code confusing. How would you feel about adding `previousMemorySize` parameter to `setGeometry` method instead?